ldes-server-infra-postgres/postgres-fragmentation-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/entity/FragmentEntity.java

```diff
@@ -0,0 +1,107 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.entity;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.FragmentPair;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.LdesFragmentIdentifier;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.TreeRelation;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.ViewName;
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.converter.FragmentPairConverter;
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.entities.Fragment;
+import jakarta.persistence.*;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@SuppressWarnings({"java:S1068", "java:S107"})
```

Comment:
A little suggestion, I don't know if we really want to use this, but I noticed that `java:S1068` is to suppress unused fields, but those fields can be annotated with `@Column` (without the need of specifying an additional name or etc.), and so could that specific warning suppression be dropped

Reply:
The warning mentions unused private fields. which is still the case